uberon_edit.obo

```diff
@@ -19,6 +19,7 @@ subsetdef: non_informative "abstract class brought in to group ontology classes
 subsetdef: organ_slim "organs, excluding individual muscles and skeletal elements"
 subsetdef: pheno_slim "Phenotype slim"
 subsetdef: phenotype_rcn "A subset specifically created for the 2012 Phenotype RCN meeting. Includes some human-specific terms that may eventually be removed when they are adequately represented with part of relationships in FMA"
+subsetdef: UBERON:8410007 "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."
```

Comment:
probably easiest to edit OBO file.  

subsetdef: UBERON:8410007 "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."
>
subsetdef:  added_for_HCA "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."